index.js

```diff
@@ -4,16 +4,17 @@ const createLogger = require('redux-logger')
 const React = require('react')
 const ReactDOM = require('react-dom')
 const { Provider } = require('react-redux')
-const { Router, createMemoryHistory } = require('react-router')
-const { syncHistoryWithStore } = require('react-router-redux')
+const { Router, createMemoryHistory, browserHistory } = require('react-router')
+const { routerMiddleware, syncHistoryWithStore } = require('react-router-redux')
 
 const config = require('./config')
 const routes = require('./routes')
 const reducer = require('./reducer')(config)
 
 const enhancer = compose(
   installReduxLoop(),
-  applyMiddleware(createLogger())
+  applyMiddleware(createLogger()),
+  applyMiddleware(routerMiddleware(browserHistory))
```

Comment:
this was a total shot in the dark. it looks like we already have history objects below, no idea how `createMemoryHistory` or `browserHistory` do or do not interoperate